randovania/games/prime1/logic_database/Chozo Ruins.txt

```diff
@@ -2354,7 +2384,13 @@ Extra - aabb: [743.8239, -217.36417, 29.29987, 834.11273, -144.96254, 81.37303]
       All of the following:
           After Hall of the Elders Unlock Doors
           Any of the following:
-              Morph Ball Bomb and Morph Ball and Bomb Jump (Expert)
+              All of the following:
+                  Morph Ball Bomb and Morph Ball
+                  Any of the following:
+                      # https://youtu.be/X48K4uhg3OY
+                      Bomb Jump (Expert)
```

Comment:
Correct me if I'm wrong, but all other instances of this trick use no more than 2 bombs. To me, complex means abusing the refill timer. If I'm right about Bomb Jump usage, I think this should be beginner/intermediate complex bomb jump

Reply:
`Bomb Jump (Expert) and Complex Bomb Jump (Beginner)`? I wouldn't leave it at beginner or intermediate since it's pretty hard to execute.

Reply:
In that case I'd rather it stay `Bomb Jump (Expert)` without the or